libs/sdk/assembly/crypto.ts

```diff
@@ -186,33 +244,72 @@ class KeyECC extends Key {
             ret[i] = value[i];
         return new PublicKey(ret);
     }
+
+    getPrivateKey(): PrivateKey {
+        const k = String.UTF8.encode(this.name, true);
+        let value = new Uint8Array(32);
+        let result = export_key_raw(k, 0, value.buffer, value.byteLength);
+        const ret: u8[] = [];
+        if (result < 0)
+            return new PrivateKey(ret); // todo : report error
+        if (result > value.byteLength) {
+            // buffer not big enough, retry with a properly sized one
+            value = new Uint8Array(result);
+            result = export_key_raw(k, 0, value.buffer, value.byteLength);
+            if (result < 0)
+                return new PrivateKey(ret); // todo : report error
+        }
+        for (let i = 0; i < result; ++i)
+            ret[i] = value[i];
+        return new PrivateKey(ret);
+    }
 }
 
 class CryptoECDSA {
 
-    static generateKey(keyName: string = ""): KeyECC {
+    static generateKey(keyName: string = "", extractable = false): KeyECC {
         const key = new KeyECC(keyName);
-        let nameBuf = String.UTF8.encode(key.name, true);
+        const nameBuf = String.UTF8.encode(key.name, true);
         generate_signing_key(nameBuf);
         return key;
     }
 
     static getKey(keyName: string): KeyECC | null {
-        let nameBuf = String.UTF8.encode(keyName, true);
+        const nameBuf = String.UTF8.encode(keyName, true);
         if (key_exists(nameBuf))
             return new KeyECC(keyName);
         return null
     }
 
+    static importKey(keyName: string = "", keyData: string, format = 'raw', extractable = false): KeyECC | null {
+        const key = new KeyECC(keyName);
+
+        const nameBuf = String.UTF8.encode(key.name, true);
+        const rawData = decode(keyData);
+
+        let iFormat = SubtleCrypto.format(format);
+        if (iFormat < 0)
+            return null;
+
+        const usages = new Uint8Array(2);
```

Comment:
This will not work, you cannot import a key that can both sign and verify. 
Currently, you have to import a private key (that can sign) and then import a corresponding public key (that can verify)
This condition is actually important:
        if (usages.includes("sign") && usages.includes("verify"))
            return false;